octomap/include/octomap/OcTreeBaseImpl.h

```diff
@@ -108,6 +111,57 @@ namespace octomap {
     inline unsigned int getTreeDepth () const { return tree_depth; }
 
     inline double getNodeSize(unsigned depth) const {assert(depth <= tree_depth); return sizeLookupTable[depth];}
+    
+    
+    // -- Tree structure operations formerly contained in the nodes ---
+   
+    /// Creates (allocates) the i-th child of the node. @return ptr to newly create NODE
+    NODE* createNodeChild(NODE* node, unsigned int childIdx);
```

Comment:
sorry for potential ignorance: could this method be made const? First impression is that no members of BaseImpl are modified? (bracing for burn)